test/test_samples_osw/upgrades/results_output.csv

```diff
@@ -0,0 +1,11 @@
+project_national-0001.osw,1,0.0,0.0,5.5,54.4,0.0,0.0,45.5,1175.9,0.0,0.0,156.9,0.0,350.2,12.0,0.0,0.0,7.6,0.0,0.0,0.0,0.0,1465.0,1327.7,0.0,0.0,0.0,0.0,-0.0,11.0,0.0,434.3,0.0,0.0,0.0,0.0,1138.0,1138.0,0.0,0.0,0.0,164.0,0.0,2262.4,3786.5,0.0,1.0,0.0,16.1,0.0,0.0,16.2,0.0,0.0,0.0,108.7,5046.0,31.3,4039.9,999.0,0.0,0.0,0.0,0.0,18.7,18.4,18.8,1.2,1.2,1.2,0.4,0.2,0.3,19.1,18.1,1.5,1.5,5396.0,1.7,0.0,7.7,0.0,12.9,30.0,5046.0,31.3,0.0,141.1,0.0,0.0,4039.9,Air Leakage,201.2,201.2,0.0,60.4,0.0
+project_testing-0001.osw,0,40.0,0.0,16.4,0.0,0.0,0.0,0.0,396.1,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,1371.7,0.0,0.0,0.0,0.0,0.0,43.4,-12466.4,11.0,0.0,0.0,0.0,159.2,0.0,2731.3,8194.0,0.0,156.6,0.0,0.0,0.0,0.0,27532.8,105902.4,0.0,1.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,-10468.7,393.5,29088.7,999.0,0.0,0.0,429.2,0.0,16.1,21.1,17.3,2.4,0.2,0.5,0.2,0.2,0.2,15.9,16.5,2.8,0.2,4287.0,3.8,3053.7,94.0,0.0,361.4,0.0,1997.8,436.0,0.0,0.0,429.2,0.0,29088.7,Air Leakage,5235.4,5614.8,1745.1,0.0,0.0
+project_national-0001.osw,1,0.0,0.0,5.5,54.4,0.0,0.0,45.5,1031.5,0.0,0.0,156.9,0.0,296.1,98.6,0.0,0.0,62.6,0.0,0.0,0.0,0.0,1465.0,1327.7,0.0,0.0,0.0,0.0,-0.0,11.0,0.0,348.3,0.0,0.0,0.0,0.0,1138.0,1138.0,0.0,0.0,0.0,409.0,33.0,2262.4,3786.5,0.0,1.0,0.0,16.1,0.0,0.0,133.1,0.0,0.0,0.0,109.1,4902.9,42.6,2500.0,17.4,0.0,0.0,0.0,0.0,18.9,18.5,18.4,1.3,1.1,1.2,0.4,0.2,0.2,19.6,17.3,1.5,1.4,5468.0,1.7,0.0,7.7,0.0,12.9,30.0,4902.9,42.6,0.0,258.3,0.0,0.0,2500.0,Refrigerator,201.2,201.2,0.0,60.4,0.0
+project_testing-0001.osw,0,40.0,0.0,16.4,0.0,0.0,0.0,0.0,408.8,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,43.3,-12466.4,11.0,0.0,348.3,0.0,159.2,0.0,2731.3,8194.0,0.0,0.0,0.0,0.0,0.0,64.0,27532.8,105902.4,0.0,1.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,-11479.5,1862.4,2500.0,17.4,0.0,0.0,1901.6,0.0,15.4,20.9,16.9,3.7,0.1,0.3,0.1,0.1,0.1,15.3,16.2,4.8,0.1,4862.0,6.6,3053.7,94.0,0.0,361.4,0.0,986.9,1905.0,0.0,0.0,1901.6,0.0,2500.0,Refrigerator,5235.4,5614.8,1745.1,0.0,0.0
+project_national-0001.osw,1,0.0,0.0,5.5,54.4,0.0,0.0,45.5,922.6,0.0,0.0,26.2,0.0,262.6,111.4,0.0,0.0,70.8,0.0,0.0,0.0,0.0,357.9,1327.7,0.0,0.0,0.0,0.0,-0.0,11.0,0.0,434.3,0.0,0.0,0.0,0.0,1138.0,1138.0,0.0,0.0,0.0,309.0,41.0,2262.4,3786.5,0.0,1.0,0.0,16.1,0.0,0.0,150.4,0.0,0.0,0.0,109.2,3629.8,40.0,853.5,25.0,0.0,0.0,0.0,0.0,15.2,17.3,15.4,0.9,0.6,0.8,0.3,0.2,0.2,16.4,16.3,1.1,0.9,5392.0,1.2,0.0,7.7,0.0,12.9,30.0,3629.8,40.0,0.0,275.7,0.0,0.0,853.5,Lighting,201.2,201.2,0.0,60.4,0.0
+project_testing-0001.osw,0,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
```

Comment:
What's going on here?

Reply:
These were failing simulations due to ill-defined upgrades.